CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v14.0.0] 2022-12-08
```

Comment:
The release v14.0.0 should include everything after https://github.com/nasa/cumulus/releases/tag/v13.4.0.  Please update CHANGELOG.md

Reply:
Updated the CHANGELOG.md as commented. Thanks Jenny.